src/components/layout.js

```diff
@@ -31,6 +38,47 @@ const Layout = ({ children }) => {
         />
         {children}
       </div>
+      <Footer
+        inputText={{
+          placeholderText: "email address",
+          widthDesktop: "291px",
+          widthMobile: "155px",
+        }}
+        footerNavigation={[
+          { name: "Programs", url: "/programs" },
+          { name: "Volunteer", url: "/new/volunteer" },
+          { name: "Donate", url: "/new/donate" },
+        ]}
+        seals={[{ logo: sealOfTransparency, alt: "Seal of Transparency" }]}
```

Comment:
This should probably have "GuideStar" written somewhere, since it's a seal from GuideStar.

Reply:
I put "GuideStar Seal of Transparency"